src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -65,27 +67,84 @@ public WebACRecipesIT() {
 
     @Before
     public void setUp() throws ClientProtocolException, IOException {
-        // TODO: this should be factored out into a repeatable method to set up each scenario
-        final HttpPost postRequest = new HttpPost(serverAddress);
+        final String credentials = "username:password";
+
+        logger.debug("setting up ACLs and authorization rules");
+
+        ingestAcl(credentials, "/acls/01/acl.ttl", "/acls/01/authorization.ttl");
+        ingestAcl(credentials, "/acls/02/acl.ttl", "/acls/02/authorization.ttl");
+        ingestAcl(credentials, "/acls/03/acl.ttl", "/acls/03/auth_open.ttl", "/acls/03/auth_restricted.ttl");
+        ingestAcl(credentials, "/acls/04/acl.ttl", "/acls/04/auth1.ttl", "/acls/04/auth2.ttl");
+        ingestAcl(credentials, "/acls/05/acl.ttl", "/acls/05/auth_open.ttl", "/acls/05/auth_restricted.ttl");
+
+        logger.debug("setup complete");
+    }
+
+    /**
+     * Convenience method to create an ACL with 0 or more authorization resources in the respository.
+     *
+     * @param credentials
```

Comment:
I'm inclined to think that there must be some way to have the IDE do this, but I have to admit that I don''t know quite how.